src/data/detail-page/tabs/backend.ts

```diff
@@ -0,0 +1,19 @@
+import { Category } from './tabsInterfaces';
+
+export const backend: Category[] = [
+  {
+    id: 'learn',
+    name: 'learn',
+    text: 'What do I learn',
+  },
+  {
+    id: 'jobs',
+    name: 'jobs',
+    text: 'Jobs',
+  },
+  {
+    id: 'resources',
+    name: 'resources',
+    text: 'Useful Resources',
+  },
```

Comment:
I have one concern here - we probably want to have the same display name, text etc. for each tab, doesn't matter in which category we are. So I would rather define in here, which tabs we need for a category, in an array with strings, for example:
`const backend = ['learn', 'jobs', 'resources']` 
And we could have one big array with objects for all of the tabs in one place.
 The problem that I see here - imagine that we decide to change the name of resources from `'Useful Resources'` to` 'Best Resources'`. In this solution we would need to change it in 6 places and we would probably prefer to change it in one place only :)

Reply:
Done :)